webapp/package.json

```diff
@@ -29,30 +29,30 @@
     "bootstrap-tour": "^0.12.0",
     "enketo-core": "4.41.6",
     "font-awesome": "^4.7.0",
-    "google-libphonenumber": "^3.2.2",
+    "google-libphonenumber": "^3.2.5",
     "gsm": "^0.1.4",
     "jquery": "3.2.x",
-    "lodash": "^4.17.11",
+    "lodash": "^4.17.15",
     "medic-nootils": "^2.0.0",
-    "messageformat": "^2.2.1",
+    "messageformat": "^2.3.0",
     "moment": "^2.24.0",
-    "mustache": "^3.0.1",
-    "ng-redux": "^4.4.0",
+    "mustache": "^3.1.0",
+    "ng-redux": "^4.4.1",
     "object-path": "^0.11.4",
     "odk-xform-compact-record-representation-for-sms": "^1.0.1",
     "openrosa-xpath-evaluator": "^1.5.0",
     "partial-json-parser": "^1.2.2",
     "pojo2xml": "^1.7.2",
     "pouchdb-browser": "^7.1.1",
     "pouchdb-debug": "^7.1.1",
-    "redux": "^4.0.1",
+    "redux": "^4.0.4",
     "redux-logger": "^3.0.6",
     "redux-thunk": "^2.3.0",
     "reselect": "^4.0.0",
-    "select2": "4.0.3",
+    "select2": "^4.0.3",
```

Comment:
So, we're bumping `select2` now? 
Webapp is still using `4.0.3` but admin uses `4.0.11`.